gulpfile.js

```diff
@@ -160,7 +160,11 @@ function buildSass() {
   };
 
   return (
-    src([`${paths.dist.theme}/*.scss`.replaceAll("//", "/")])
+    src(
+      [
+        `${paths.dist.theme}/**/*.scss`.replaceAll("//", "/"), 
+        `${paths.src.projectSass}/**/*.scss`.replaceAll("//", "/")
+      ])
```

Comment:
@mejiaj @thisisdano I noticed that `paths.src.projectSass` was only referenced in the `watchSass` task and not included in the `buildSass` task. When I add `projectSass` to the src in `buildSass`, I am able to see the files successfully compile inside `paths.dist.css`. 

I wanted to confirm:
1. Is there any risk to adding `projectSass` in the `buildSass` task? Was its exclusion intentional or an oversight?
1. Is what we see in the screenshot below the intended result? 
    1. Please note: if the user has a styles.scss file at the root of their projectSass directory, it will not compile but theme/styles.scss will

Please let me know if you have questions. 

![image](https://user-images.githubusercontent.com/93996430/163583810-bf7b68e2-c825-4361-b1f0-ceb17df11ea1.png)

Reply:
1. I think it's reasonable to include in `buildSass` if it was in `watchSass`
2. That's what I would expect
   1. This sounds fine since it'd be a duplicate file